### ISSUE TITLE
Seeking Advice for Slow Website Speed

### ISSUE THREAD

**Issue by albertsmith7175** [albertsmith7175/gitub-with-website#22]
Lately, I've been encountering a frustrating issue with my [wall decals](https://nvsvisuals.com/wall-vinyl-wraps/) website – it's loading painfully slowly. Every click feels like an eternity, and I'm worried it might be turning visitors away. I've tried a few basic optimizations like compressing images and minifying CSS, but it doesn't seem to be making much of a difference. I'm reaching out to the community for advice – have any of you encountered similar problems with sluggish website performance, and if so, what strategies or tools did you employ to rectify it? Any insights or suggestions would be greatly appreciated. Thanks in advance for your help!

**Comment by SOCIALXAT** [comment]
Yes I have encountered a similar situation with my [property maintenance London](https://lgcgroup.london) website. I am also looking for solutions
